source/modules/home/screens/FeedScreen/index.tsx

```diff
@@ -2,14 +2,26 @@ import { NativeStackNavigationOptions } from '@react-navigation/native-stack'
 import { Button } from '@rneui/themed'
 import React from 'react'
 import { useLogout } from 'source/core/hooks'
-import { Screen, Text } from './styles'
+import { FeedPost } from '../../components'
+import { Screen } from './styles'
 
 export const FeedScreen = () => {
   const { doLogout } = useLogout()
 
   return (
     <Screen testID="feed-screen-test-id">
-      <Text>FeedScreen</Text>
+      <FeedPost
+        type="pet_lost"
+        description="Testando"
+        postImage="https://i.imgur.com/4zSAWJ5.jpeg"
+        user={{
+          photo: 'https://i.imgur.com/YcP0tik.jpeg',
+          full_name: 'João Pessoa',
+        }}
+        onPressComment={() => console.log('comentário')}
+        onPressDenuncie={() => console.log('denuncia')}
+        onPressPostDetails={() => console.log('detalhes')}
+      />
       <Button title="Deslogar" onPress={() => doLogout()} />
```

Comment:
aqui você pode chamar o doLogout diretamente assim:
```suggestion
      <Button title="Deslogar" onPress={doLogout} />
```